containers/svclone/svclone_config.ini

```diff
@@ -0,0 +1,157 @@
+##################################################################################################################
+# Sample config file for SVclone
+##################################################################################################################
+
+# SV processing-related options
+
+[BamParameters]
```

Comment:
Would like to discuss the necessity of parameterizing these configs.

Reply:
Sure. A short answer, the svclone pipeline requires input for `insert_mean` and `insert_std`. If you add `-1` it is supposed to infer dynamically and this part often breaks. I'm not sure what happens if you assume a fixed insert mean and standard deviation. The configuration for the naming conventions of the SV classes is also modified to fit our inputs.

Reply:
@gongyixiao i think we resolved this discussion before. let me know if i can close it.